_non-cosmos/internetcomputer/assetlist.json

```diff
@@ -36,6 +36,41 @@
         "website": "https://internetcomputer.org/",
         "twitter": "https://x.com/dfinity"
       }
+    },
+    {
+      "description": "A multi-chain bitcoin twin, trustlessly created by chain-key cryptography and Internet Computer smart contracts that directly hold raw bitcoin.",
+      "extended_description": "Chain-key Bitcoin (ckBTC) is an ICRC-2-compliant token that is backed 1:1 by BTC held 100% on the mainnet. ckBTC does not rely on a centralized bridge to facilitate the conversion between BTC and ckBTC, which makes it substantially more secure when compared to other traditional 'wrapped' tokens.",
+      "denom_units": [
+        {
+          "denom": "uckBTC",
+          "exponent": 0
+        },
+        {
+          "denom": "ckBTC",
+          "exponent": 8
+        }
+      ],
+      "type_asset": "unknown",
+      "base": "e8s",
+      "name": "ckBTC",
```

Comment:
This name is acceptable, but I thought this would be something like "Chian Key BTC"?